.travis.yml

```diff
@@ -6,7 +6,9 @@ matrix:
 
```

Comment:
Add these two lines at the head of .travis.yml and try again to compile correctly: 

```
sudo: required
dist: trusty
```